account_move_report/views/account_move_report.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="account_entries_report" name="Entries Report">
+        <t t-call="web.html_container">
+            <t t-call="web.external_layout">
+                <t t-foreach="docs" t-as="o">
+                    <div class="page">
+                        <div class="oe_structure"/>
+                        <br/>
+                        <div class="row">
+                            <div class="col-xs-6">
```

Comment:
Please change `col-xs-6` for only `col-6` 

To get this:
![Screen Shot 2019-03-22 at 9 45 35 AM](https://user-images.githubusercontent.com/7606656/54835187-4a28e200-4c87-11e9-8215-9be2f8b5835b.png)